plugins/scaffolder-backend-module-github/src/actions/github.test.ts

```diff
@@ -41,6 +41,7 @@ import {
   GithubCredentialsProvider,
   ScmIntegrations,
 } from '@backstage/integration';
+import { when } from 'jest-when';
```

Comment:
Think you can remove this, we dropped this library from the scaffolder tests recently.

Reply:
Removed, apologies, this got in from a bad rebase.